src/runtime_src/core/tools/common/XBUtilities.cpp

```diff
@@ -384,7 +384,8 @@ XBUtilities::get_available_devices(bool inUserDomain)
 
     //if factory mode
     if (is_mfg) {
-      std::string vbnv = "xilinx_" + xrt_core::device_query<xrt_core::query::board_name>(device) + "_GOLDEN";
+      unsigned int mGoldenVer = xrt_core::device_query<xrt_core::query::mfg_ver>(device);
```

Comment:
minor comment, for future: please use `auto` instead of specifying the exact type. Let the function determine what should be the type of the variable